sys/kern/kern_linker.c

```diff
@@ -1387,7 +1387,9 @@ sys_kldsym(struct thread *td, struct kldsym_args *uap)
 	    lookup.symname, &lookup.symvalue, &lookup.symsize);
 	if (error != 0)
 		return (error);
-	error = copyout(&lookup, uap->data, sizeof(lookup));
+	error = copyout(&lookup.symvalue, (char * __capability)uap->data +
```

Comment:
Needs `__unbounded_addressof` for the purecap kernel?

Reply:
Since `struct kld_sym_lookup` has only two members that need copying I'd be tempted to use two `suword` calls here.